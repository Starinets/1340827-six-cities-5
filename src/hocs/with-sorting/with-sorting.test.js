import React from 'react';
import renderer from 'react-test-renderer';
import withSorting from './with-sorting';
import {
  emptyFunction
} from '../../test-data/test-data';

const MockComponent = () => {
  return (
    <div/>
  );
};

const MockComponentWrapped = withSorting(MockComponent);

it(`HOC rendered correctly -> withSorting`, () => {
  const tree = renderer.create(
      <MockComponentWrapped
        isOpen={ true }
        onListItemClick={ emptyFunction }
        onListNameClick={ emptyFunction }
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
