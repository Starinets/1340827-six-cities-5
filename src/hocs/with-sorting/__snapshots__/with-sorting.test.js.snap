// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HOC rendered correctly -> withSorting 1`] = `<div />`;
